sdk/java/Makefile

```diff
@@ -1,11 +1,20 @@
-# Builds the project and runs unit tests.
+# Builds the project and runs fast unit tests.
 build::		ensure
 	gradle --console=plain build
 
 ensure::
 	java --version
 	gradle --version
 
+# Builds the project and runs all unit tests, both fast and auto tests.
+# A coverage report is generated at pulumi/build/reports/jacoco/allTests
+test_all::		ensure
```

Comment:
Should all these targets be marked `.PHONY`?  It probably doesn't matter now, but if someone accidentally has a weird file in their repo for whatever reason they might get confused. (This applies to the existing targets as well, but I just noticed it now)

Reply:
Yep. Done.